src/vms/pvm/etna-builder/spend.ts

```diff
@@ -37,6 +47,20 @@ type SpendResult = Readonly<{
 }>;
 
 export type SpendProps = Readonly<{
+  /**
+   * List of addresses that are used for change outputs.
+   *
+   * Defaults to the addresses provided in `fromAddressesBytes`.
+   */
+  changeAddressesBytes?: readonly Uint8Array[];
+  /**
+   * Optionally specifies the output owners to use for the unlocked
+   * AVAX change output if no additional AVAX was needed to be burned.
+   * If this value is `undefined` or `null`, the default change owner is used.
+   *
+   * Used in ImportTx.
+   */
+  changeOwnerOverride?: OutputOwners | null;
```

Comment:
Can `changeOwnerOverride` and `changeAddressesBytes` be consolidated? Not sure why you need to specify change owners via 2 parameters. Is having 2 parameters an API oversight we are inheriting or is there a legitimate usecase for over specifying 2 different change owners and having `changeAddressesBytes` overridden?

Reply:
Changes in e0835e4 should resolve.